citar-org-roam.el

```diff
@@ -40,6 +40,17 @@
   :group 'citar-org-roam
   :type 'string)
 
+(defcustom citar-org-roam-capture-template-key
+  nil
+  "When non-nil, `citar-org-roam--create-capture-note' will use the
+template associated with the key in `org-roam-capture-templates'.
```

Comment:
Thanks for this!

On this part, is it feasible that a user might have more than one template, that they would want to choose from?

If yes, perhaps this should be a list, and if the length is one, can bypass the menu?

I have a list of some UX questions like these on #2 (see also #11; if we can close both of these with this PR you should add a "close" line to the commit message).

Reply:
Thanks @bdarcus , that's definitely very interesting! At the moment, I access my templates by overwriting the `orb-citar-source` list, calling `orb-citar-edit-note` to create a new note. I don't know if that could be of any help for the implementation of the code.

Reply:
I apologize for not reading on #2 and #11 before submitting this PR. Seems to me like @bdarcus is aiming to extend org roam's capture functionality like orb? What I added was pretty much just passing a `:keys` argument to `org-roam-capture-` so that it uses the template I like. I don't see how this commit will help with that.

Reply:
@c1-g could you please add a simple config example, with template, ideally to the README, or at least in a comment here?

I was looking to test this but realizing I don't know how :-)